lib/s3routes/routesUtils.ts

```diff
@@ -18,6 +23,14 @@ export type CallApiMethod = (
     callback: (err: ArsenalError | null, ...data: any[]) => void,
 ) => void;
 
+export type CallPostObject = (
+    request: http.IncomingMessage,
+    response: http.ServerResponse,
+    log: RequestLogger,
+    callback: (err: ArsenalError | null, ...data: any[]) => void,
+) => void;
+
+
```

Comment:
you could keep the same `CallApiMethod` type for the post so it stays consistent for everything

Reply:
I suppose, if we agree the methodName can just be ignored which is why I made the change